app/src/main/res/layout/ongoing_topic_item.xml

```diff
@@ -67,7 +67,7 @@
           android:gravity="start"
           android:lines="2"
           android:text="@{viewModel.topicTitle}"
-          android:textColor="@color/color_def_white"
+          android:textColor="@color/component_color_shared_topic_card_primary_text_view_color"
```

Comment:
Update the name of `component_color_shared_topic_card_primary_text_view_color` to `component_color_shared_topic_card_text_view_color`

Basically remove `primary` because we are actually using it in `Topic Title` (which is primary text) and also `Story Count` (which is secondary text).

Reply:
Done